site/content/blog/2022-03-01-whats-new-in-svelte-march-2022.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "What's new in Svelte: March 2022"
+description: "Svelte Summit Spring is coming... and page endpoints are here!"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Just announced: [Svelte Summit Spring](https://www.sveltesummit.com/) will be taking place on April 30, 2022. The 5th Virtual Svelte Conference is [looking for speakers](https://www.sveltesummit.com/#speakers) and [sponsors](https://www.sveltesummit.com/sponsors)... so it's time to dust off those proposals!
+
+Also, some long-requested features were added to SvelteKit this month... including page endpoints! This change in how `load` functions makes it easier to fetch data required for basic pages, redirect from POST responses and handle 404s and other errors.
+
+More on that and other new features and fixes below!
+
+## What's new in SvelteKit
```

Comment:
- new searchable multipage docs with type definitions and hoverable code examples
- option to setup Playwright for testing (https://github.com/sveltejs/kit/pull/4056)